.github/workflows/tests-arm64-template.yaml

```diff
@@ -40,19 +56,19 @@ jobs:
           export JUNIT_REPORT_DIR=$(realpath ${TARGET})
           case "${TARGET}" in
             linux-arm64-integration-1-cpu)
-              make gofail-enable
-              GOOS=linux GOARCH=arm64 CPU=1 make test-integration
+              if [ "${{ inputs.gofailMake }}" == "yes" ]; then make gofail-enable; fi
```

Comment:
I'm worried that such forced compatibility will make it harder to add/update tests on main branch and will not help at all with preventing a breaking change. 

I think I would just to prefer to keep the definition of  periodic and pull request workflows separate.

Reply:
These templates are only used for periodic.  We don't run arm64 workflows on pull request yet.

I do share your concern around this conditional hack though.